internal/resources/resource_host.go

```diff
@@ -220,6 +221,12 @@ func hostSchema() map[string]*schema.Schema {
 			Optional:    true,
 			Description: "map of label name to label value for this host",
 		},
+		hISCSIDiscoveryIP: {
+			Type:        schema.TypeString,
+			Optional:    true,
```

Comment:
Remove "Optional" since it is "Computed".

Reply:
Removed optional